pkg/app/src/components/CampaignsSection/CampaignsList/campaignsList.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback } from 'react'
+
+import AddIcon from '@mui/icons-material/Add'
+import { Box, Button, Card, Grid, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { useTranslation } from 'react-i18next'
+import { CampaignStatus } from 'src/@types/campaignStatus'
+import { Campaign } from 'src/queries'
+
+import CampaignCard from './campaignCard'
+import LoadingCampaignCard from './loadingCampaignCard'
+
+export const PlusIcon = () => {
+	const theme = useTheme()
+	return (
+		<Box
+			sx={{
+				borderRadius: theme.shape.borderRadiusLg,
+				maxWidth: 56,
+				maxHeight: 56,
+				padding: 2,
+				backgroundColor: theme.palette.grey[900],
+			}}
+		>
+			<AddIcon sx={{ color: theme.palette.grey[600] }} />
+		</Box>
+	)
+}
+
+interface ComponentProps {
+	data: Campaign[]
+	loading: boolean
+	title: boolean
+	isAdmin: boolean
+	onCreateCampaignClicked?: () => void
+}
+
+export function CreatedCampaignSection({ data, loading, title, isAdmin, onCreateCampaignClicked }: ComponentProps) {
```

Comment:
Please rename the component to "CampaignsList" it should correspond with the filename. (Would also solve the problem with the misleading name)

Reply:
yes noted